pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -51,15 +51,16 @@ const (
 
 // Roles
 const (
-	ClusterAdminRoleName    = "cluster-admin"
-	SudoerRoleName          = "sudoer"
-	ClusterReaderRoleName   = "cluster-reader"
-	AdminRoleName           = "admin"
-	EditRoleName            = "edit"
-	ViewRoleName            = "view"
-	SelfProvisionerRoleName = "self-provisioner"
-	BasicUserRoleName       = "basic-user"
-	StatusCheckerRoleName   = "cluster-status"
+	ClusterAdminRoleName       = "cluster-admin"
+	SudoerRoleName             = "sudoer"
+	ClusterReaderRoleName      = "cluster-reader"
+	AdminRoleName              = "admin"
+	EditRoleName               = "edit"
+	ViewRoleName               = "view"
+	SelfProvisionerRoleName    = "self-provisioner"
+	BasicUserRoleName          = "basic-user"
+	StatusCheckerRoleName      = "cluster-status"
+	SelfAccessReviewerRoleName = "self-access-reviewer"
```

Comment:
because all the APIs that check access are `*Review` (`SubjectAccessReview`, etc)